src/clr-angular/button/button-loading/loading-button.ts

```diff
@@ -5,26 +5,91 @@
  */
 
 
-import {Component} from "@angular/core";
-import {LoadingListener} from "../../utils/loading/loading-listener";
-
+import {animate, keyframes, style, transition, trigger} from "@angular/animations";
+import {Component, ElementRef, EventEmitter, Output, Renderer2} from "@angular/core";
+import {ClrLoadingState, LoadingListener} from "../../utils/loading";
 
 @Component({
     selector: "button[clrLoading]",
     template: `
-        <span class="spinner spinner-inline" *ngIf="loading"></span>
-        <ng-content></ng-content>
+        <ng-container [ngSwitch]="state">
+            <span *ngSwitchCase="buttonState.LOADING">
+                <span @spinner class="spinner spinner-inline spinner-loading"></span>
```

Comment:
What does the `.spinner-loading` class do and where is it declared? I dont see it in the CSS.

Reply:
Hmm.. maybe a class I was using while implementing and no longer needed. Will doublecheck and remove if that's the case.